### PR TITLE
Nova 4 Compatibility

### PR DESCRIPTION
**Pull request by mertasan**

### CODE REVIEW

**Comment by Tarpsvo** [comment]
Heya! Did you test it as well?

**Comment by mertasan** [comment]
No, I haven't tested it. I'm still trying to install Nova. If you don't have time to test it, I will test it asap.

**Comment by Gummibeer** [comment]
Hey, have tested the branch in one of my projects and it hasn't rendered any drag-sortable icon like before. So I assume that there's more to do to make it work with Nova v4!?

**Comment by mertasan** [comment]
@Gummibeer I guess Vue files need to be made compatible with Vue 3.

I didn't have time to do this. But I will.

**Comment by SanjinOsmanbasic** [comment]
Really appreciate the time you are putting in to this update. 🙏

**Comment by aromer** [comment]
I'm very interested in this too. I have a working Nova 4 instance and can possibly assist if needed.

**Comment by maloun96** [comment]
any updates?

**Comment by anthony-brindley** [comment]
Super eager for this!

**Comment by wasitum** [comment]
Extremely excited for this, any update by any chance ?

**Comment by ricklambrechts** [comment]
Nova loads their components after loading other packages. So I think that it is not possible to overwrite the ResourceTableRow and ResourceTable component at the moment.
@taylorotwell Could you look into adding support for this?

File: resources/js/tool.js [hunk 1]
```diff
@@ -2,10 +2,10 @@ import ResourceTable from './components/ResourceTable';
 import ResourceTableRow from './components/ResourceTableRow';
 import ReorderButtons from './components/ReorderButtons';
 
-Nova.booting((Vue, router, store) => {
-  Vue.component('resource-table', ResourceTable);
-  Vue.component('resource-table-row', ResourceTableRow);
-  Vue.component('reorder-buttons', ReorderButtons);
+Nova.booting((app, router, store) => {
```
Comment (crynobone) [inline resources/js/tool.js:5]:
```suggestion
Nova.booting((app, store) => {
```

File: package.json [hunk 2]
```diff
@@ -8,18 +8,18 @@
...
     "prettier": "^2.5.1",
     "resolve-url-loader": "^4.0.0",
     "sass": "^1.49.9",
     "sass-loader": "^12.6.0",
-    "vue-loader": "^15.9.8",
+    "vue-loader": "^16.8.3",
     "vue-template-compiler": "^2.6.14"
   },
   "dependencies": {
     "laravel-nova": "^1.12.3",
```
Comment (crynobone) [inline package.json:21]:
```suggestion
```

**Comment by s-fog** [comment]
Really waiting for this update :) Cant use Nova 4 because of it :(

**Comment by LTKort** [comment]
Just did a new PR for this #121, tested and working when using the fix of https://github.com/laravel/nova-issues/issues/4143#issue-1221940614

**Comment by kbellpostman** [comment]
> Just did a new PR for this #121, tested and working when using the fix of [laravel/nova-issues#4143 (comment)](https://github.com/laravel/nova-issues/issues/4143#issue-1221940614)

Yess! Fingers crossed :)

**Pull request merged**